### ISSUE TITLE
Duplicate presence records

### ISSUE THREAD

**Issue by Worrelpa** [stuartridout/teamlink#43]
Not sure if anyone has seen an issue where presence is when changed is logging 3 presence records which is knocking off the confirmed number in the site view. As it is counting the rows and getting more than people showing up in the onsite view.  

Going to try to work around it by filtering the count on unique records by email for each user mail id. Though I am not sure how to pick up the latest record for the count to be correct.

**Comment by stuartridout** [comment]
What version of the app are you on?On 26 Oct 2022, at 13:16, Worrelpa ***@***.***> wrote:﻿
Not sure if anyone has seen an issue where presence is when changed is logging 3 presence records which is knocking off the confirmed number in the site view. As it is counting the rows and getting more than people showing up in the onsite view.
Going to try to work around it by filtering the count on unique records by email for each user mail id. Though I am not sure how to pick up the latest record for the count to be correct.

**Comment by Worrelpa** [comment]
Latest 10038

We have modified our instance some what but not that part of it. 
We added / hid a few options but not changed the core of it.

I am wondering if someone is clicking multiple times on the icon before the first patch has gone through. So it creates new records each time. Thinking of trying a disable the part and then enable once the processing is done to avoid that, but that ony a guess at what is happening. Also will look at getting distinct records for the site count.
